lib/core/include/rodsErrorTable.h

```diff
@@ -770,6 +770,7 @@ NEW_ERROR(INVALID_KVP_STRING,                   -1826000)
 NEW_ERROR(PLUGIN_ERROR_MISSING_SHARED_OBJECT,   -1827000)
 NEW_ERROR(RULE_ENGINE_ERROR,                    -1828000)
 NEW_ERROR(REBALANCE_ALREADY_ACTIVE_ON_RESOURCE, -1829000)
+NEW_ERROR(INVALID_RESOURCE_HIERARCHY,           -1830000)
```

Comment:
could be convinced this needs a new error type... or that we could use `HIERARCHY_ERROR` as is...